plugin/src/sbt-test/paradox/libraryDependency/test

```diff
@@ -0,0 +1,9 @@
+# check that the generated pom does not contain the paradoxTheme as dependency
+> publishLocal
+
+# ensure that our test is valid, check that this grep is successful
+$ exec grep "<artifactId>scala-library" target/scala-2.12/librarydependencytest_2.12-0.1.0-SNAPSHOT.pom
```

Comment:
Tests are failing because they're run with 2.10 AFAICS. Unfortunately I couldn't get it to work with wildcards - is there any way to write it in more generic way?